services/catalog/rs_server_catalog/user_catalog.py

```diff
@@ -428,8 +427,8 @@ async def manage_get_response(
         if "detail" not in content:  # Test if the user is authenticated.
             if request.scope["path"] == "/":  # /catalog
                 try:
-                    api_key = request.headers["x-api-key"]
-                    auth_roles, _, user_login = await apikey_security(request, api_key)
+                    auth_roles = request.state.auth_roles
```

Comment:
@cyouri-csgroup the authentication info is already in the request.state. We must not call apikey_security manually.